kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -130,6 +130,17 @@ function _getCurrentChannelRootTopicId() {
   });
 }
 
+/**
+ * Action inhibition checks
+ *
+ * These generator functions produce functions that help to determine whether the
```

Comment:
Optimistic commenting. Assuming that we will have more of these in the future (also, this should probably end up in the core if we want to use this across several apps).